src/components/landing/listing.js

```diff
@@ -0,0 +1,21 @@
+import Link from "next/link";
+import Image from "next/image";
+
+function Listing({ data }) {
+  return (
+    <>
+      <article className="listing">
+        <div className="listing__image">
```

Comment:
Wouldn't this class style be confused with the image style itself?

Reply:
I have change the class Name from "listing__image" to "listing__image__container"